Sources/Model/ObjC/TracksDeviceInformation.m

```diff
@@ -2,7 +2,7 @@
 
 #if TARGET_OS_IPHONE
 #import <UIKit/UIKit.h>
-#import <UIDeviceIdentifier/UIDeviceHardware.h>
+@import UIDeviceIdentifier;
```

Comment:
UIDeviceIdentifier accepted my change, and now directly supports Swift Package Manager. This is the right way to import it when using SPM.